src/main/java/helpers/DataHelper.java

```diff
@@ -0,0 +1,51 @@
+package helpers;
+
+import com.github.javafaker.Faker;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Random;
+
+public class DataHelper {
+    private static Faker faker = new Faker();
+    private static Random random = new Random();
+
+    public static String getRandomEmail() {
+        return faker.internet().emailAddress();
+    }
+
+    public static String getRandomText(int number) {
```

Comment:
number of what?
should be length or numberOfChar

Reply:
Thank you. I fixed it.